src/components/CromwellModal.js

```diff
@@ -23,21 +23,21 @@ const titleId = 'cromwell-modal-title'
 export const CromwellModalBase = withDisplayName('CromwellModal')(
   ({
     onDismiss, onSuccess, apps, appDataDisks, workspace, workspace: { workspace: { namespace, bucketName, name: workspaceName, googleProject } },
-    isAnalysisMode = false
+    shouldHideCloseButton = false
```

Comment:
There is no reason for this default value to be false-- the CromwellModal is only ever shown in the Analyses tab (AFAIK). 

As we chatted, I think that the default value of `shouldHideCloseButton` should be true, and it should only be specified as an option in the very rare cases where it should be false (cases which are going away soon anyway).

Part of my confusion here is that I expected this "close button" to be the dialog close button, as I'm only used to seeing these modals within the parent dialog (from the new Analyses tab). I am wondering if there is also a clearer name… perhaps `hideInnerCloseButton`? I'm not crazy about that, but with the current name I was quite confused.

Reply:
It'll help a lot to make the changes where it only appears in the legacy cases, and it should eventually be phased out (and not be needed in any new code). That should eliminate most confusion I think :)